src/main/java/Tasks.java

```diff
@@ -0,0 +1,45 @@
+import java.util.ArrayList; // import the ArrayList class
+
+public class Tasks {
+    private ArrayList<Task> tasks_;
```

Comment:
is there a reason why you placed undescore at the back of the variables? seems redundant